src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala

```diff
@@ -222,7 +251,10 @@ final case class AgreementApiServiceImpl(
       eService       <- catalogManagementService.getEServiceById(agreement.eserviceId)
       _              <- verifyCloningConflictingAgreements(requesterOrgId, agreement.eserviceId)
       consumer       <- tenantManagementService.getTenant(requesterOrgId)
-      _              <- validateCertifiedAttributes(eService, consumer).whenA(eService.producerId != consumer.id)
+      descriptor     <- eService.descriptors
+        .find(_.id == agreement.descriptorId)
+        .toFuture(DescriptorNotFound(eService.id, agreement.descriptorId))
+      _              <- validateCertifiedAttributes(descriptor, consumer).whenA(eService.producerId != consumer.id)
       newAgreement   <- agreementManagementService.createAgreement(agreement.toSeed)
       documents      <- createAndCopyDocumentsForClonedAgreement(agreement, newAgreement)
```

Comment:
[here](https://github.com/pagopa/interop-be-agreement-process/pull/233/files#r1239525784)